test/build-test-images.sh

```diff
@@ -22,8 +22,10 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-source $(dirname "${BASH_SOURCE[0]}")/build-utils.sh
-source $(dirname "${BASH_SOURCE[0]}")/init-buildx.sh
+source "$(dirname "${BASH_SOURCE[0]}")/build-utils.sh"
```

Comment:
`basedir="$(dirname "${BASH_SOURCE[0]}")"`? Not really happy to see the dirname call three times.